AdobeStockImageAdminUi/view/adminhtml/web/js/components/grid/column/image-preview.js

```diff
@@ -7,11 +7,11 @@ define([
     'jquery',
     'knockout',
     'Magento_Ui/js/grid/columns/column',
-    'Magento_AdobeIms/js/action/authorization',
+    'Magento_AdobeIms/js/action/signIn',
```

Comment:
Image preview should not be dependent on the component responsible for displaying user profile information should be independent. Please make `signIn` component an independent component in AdobeStockAdminUi module that is relying on data provided by AdobeIms module.

Reply:
ok